packages/chat-stateful-client/src/modifiers/ClearErrors.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { ChatClientState, ChatErrorTargets } from '../ChatClientState';
+import { ChatStateModifier } from '../StatefulChatClient';
+
+export const newClearErrorsModifier = (targets: ChatErrorTargets[]): ChatStateModifier => {
```

Comment:
I am curious if this can just be called clearErrorsModifier?

Reply:
It is a factory function that returns a modifier. I generally follow the `new*` naming pattern for factory functions.

But don't have very strong opinions if others think that the `new` is unnecessary.

Reply:
ok so this is really creating an instance of a modifier meant for clearing specific errors. ok sure i think this should be fine.